examples/browser-api-playground/src/types.ts

```diff
@@ -1,3 +1,5 @@
+import type { Options } from '@gleanwork/web-sdk'
+
 export const enum AuthType {
```

Comment:
As discussed offline, should we remove `AuthOptions` and ask QA to pass the authToken (which can be either an anonymous token or a server-side token)?

Reply:
Yeah, that sounds good to me.